canonical/ubuntu.com
# Update names for Cred

## Done

- Changed URLs from "credentialling" to "credentials" and added appropriate redirects

Fixes https://warthogs.atlassian.net/browse/CRED-214, https://warthogs.atlassian.net/browse/CRED-216

## QA

Redirects:
- Verify that [/credentialling](https://ubuntu-com-12243.demos.haus/credentialling) redirects to [/credentials](https://ubuntu-com-12243.demos.haus/credentials)
- Verify that [/credentialling/syllabus](https://ubuntu-com-12243.demos.haus/credentials/syllabus) redirects to [/credentials/syllabus](https://ubuntu-com-12243.demos.haus/credentials/syllabus)
- Verify that [/credentialling/self-study](https://ubuntu-com-12243.demos.haus/credentials/self-study) redirects to [/credentials/self-study](https://ubuntu-com-12243.demos.haus/credentials/self-study)

Exam names:
- Verify that the microcert exams have been renamed to CUE: Linux, CUE: Desktop, and CUE: Server on the [main page](https://ubuntu-com-12243.demos.haus/credentials), [syllabus](https://ubuntu-com-12243.demos.haus/credentials/syllabus), and [study guide](https://ubuntu-com-12243.demos.haus/credentials/self-study)

## Screenshots

![cue-naming-update](https://user-images.githubusercontent.com/995051/199200614-ee6260fd-133b-49f0-94e8-3bdab40bf9f0.png)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
